src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/SetAzureSqlDatabaseAuditingPolicy.cs

```diff
@@ -148,16 +161,16 @@ private void ApplyUserInputToTableAuditingModel(DatabaseAuditingPolicyModel mode
                 ModelAdapter.ClearStorageDetailsCache();
             }
             if (!string.IsNullOrEmpty(StorageKeyType))
```

Comment:
Hi Adam,

we don't want AuditType to be a switch parameter, because it is a breaking change. please read Alex Umansky mail regarding this issue.
I have added you to the code review.

thanks,
Ayelet